src/GlobalPayments.Api/Gateways/BillPayProvider.cs

```diff
@@ -19,6 +20,8 @@ internal class BillPayProvider: IBillingProvider, IPaymentGateway, IRecurringSer
 
         public string ServiceUrl { get; set; }
 
+        public IWebProxy WebProxy { get; set; }
```

Comment:
This is a very useful enhancement, in this case you will have to send the webproxy to each of the gateways to make it work: AuthorizationRequest, ManagementRequest, BillingRequest and RecurringRequest.